_config.yml

```diff
@@ -55,806 +55,810 @@ plugins:
+        url: "/sdk/ios/analytics"
+        icon: "/images/icons/Misc/Cloud.svg"
+        services:
+          - title: Amazon Pinpoint,
+            url: https://aws.amazon.com/pinpoint/
+          - title: Amazon Kinesis,
+            url: https://aws.amazon.com/kinesis/
+          - title: Custom Plugin
+            url: https://aws-amplify.github.io/amplify-js/media/analytics_guide#using-a-custom-plugin
+      - title: Push Notifications
+        excerpt: Push notifications with campaign analytics and targeting.
+        cta: Read more
+        class: aws-icon-ps-60-device-message
+        url: "/sdk/ios/push-notifications"
+        icon: "/images/icons/Misc/Cloud.svg"
+        services:
+          - title: Amazon Pinpoint
+            url: https://aws.amazon.com/pinpoint/
+      - title: PubSub
+        excerpt: Connect your app to message-oriented middleware on the cloud.
+        cta: Read more
+        class: aws-icon-ps-60-circular-circuit
+        url: "/sdk/ios/pubsub"
+        icon: "/images/icons/Misc/Cloud.svg"
+        services:
+          - title: AWS IoT,
+            url: https://aws.amazon.com/iot/
+          - title: Generic MQTT,
+            url: https://docs.aws.amazon.com/iot/latest/developerguide/protocols.html#mqtt
+          - title: Custom Plugin
+            url:
 
-    - category: Manual SDK Configuration
-      title: SDK Configuration
-      excerpt: Low level config
-      base_url: '/ios/manualsetup'
-      subs:
-        - title: SDK Setup Options
-          excerpt: Setup Options for the SDK
-          cta: Read more
-          class: aws-icon-ps-60-circular-circuit
-          url: '/ios/manualsetup'
-    - category: Docs version
-      title: Docs version
-      excerpt: Version of the docs
-      version: amplify1.0
-      ##base_url: '/ios/start'
-      subs:
-        - title: Amplify iOS
-          excerpt: Version of the docs
-          cta: Read more
-          class: aws-icon-ps-60-circular-circuit
-          url: '/ios/start'
-        - title: iOS SDK 2.0
-          excerpt: Version of the docs
-          cta: Read more
-          class: aws-icon-ps-60-circular-circuit
-          url: '/sdk/ios/start'
+  - category: Manual SDK Configuration
+    title: SDK Configuration
+    excerpt: Low level config
+    base_url: "/ios/manualsetup"
+    subs:
+      - title: SDK Setup Options
+        excerpt: Setup Options for the SDK
+        cta: Read more
+        class: aws-icon-ps-60-circular-circuit
+        url: "/ios/manualsetup"
+  - category: Docs version
+    title: Docs version
+    excerpt: Version of the docs
+    version: amplify1.0
+    ##base_url: '/ios/start'
+    subs:
+      - title: Amplify iOS
+        excerpt: Version of the docs
+        cta: Read more
+        class: aws-icon-ps-60-circular-circuit
+        url: "/ios/start"
+      - title: iOS SDK 2.0
+        excerpt: Version of the docs
+        cta: Read more
+        class: aws-icon-ps-60-circular-circuit
+        url: "/sdk/ios/start"
```

Comment:
Seeing that most of these changes look like prettier doing something, do we want to do the same config change we did for JS last night?

Reply:
There is no prettier in docs v1 so I don't believe so. Not sure why my VS code formatted everything..

Reply:
I think this is valid formatting though

Reply:
Agreed it was more of a callout if it was something with Prettier, which it isn't, and also just seeing what it was.

Reply:
This was non-blocking btw